Src/PerformanceCollector/Shared/Implementation/PerformanceCounterStructure.cs

```diff
@@ -0,0 +1,38 @@
+﻿namespace Microsoft.ApplicationInsights.Extensibility.PerfCounterCollector.Implementation
+{
+    /// <summary>Represents basic performance counter structure.</summary>
```

Comment:
If you need one - you probably doing something wrong. What we need is `reportAs` name and a class the capable extracting counters. There should be no reason to keep around the original category, counter name and instance names.